README.md

```diff
@@ -57,15 +57,17 @@ The plugin currently provides the following metrics:
-| FallbackThrottleFactorApplied | Counter     | The number of times the plug-in has transitioned to using the fall back factor | ThrottleFactor | `observingBrokerId`                           |
-| LimitViolated                 | Counter     | A count of the number `logDir`s which violate the configured limit             | ThrottleFactor | `observingBrokerId`                           |
-| ActiveBrokers                 | Gauge       | The current number of brokers returned by the describeCluster rpc              | VolumeSource   | `observingBrokerId`                           |
-| ActiveLogDirs                 | Gauge       | The number of logDirs returned by the describeLogDirs RPC                      | VolumeSource   | `observingBrokerId`                           | 
-| AvailableBytes                | Gauge       | The number of available bytes returned by the describeLogDirs RPC              | VolumeSource   | `[observingBrokerId, remoteBrokerId, logDir]` |
-| ConsumedBytes                 | Gauge       | The number of consumed bytes returned by the describeLogDirs RPC               | VolumeSource   | `[observingBrokerId, remoteBrokerId, logDir]` |
+| Name                          | Metric Type | Meaning                                                                        | Type                  | Tags                                          |
+|-------------------------------|-------------|--------------------------------------------------------------------------------|-----------------------|-----------------------------------------------|
+| ThrottleFactor                | Gauge       | The current factor applied by the plug-in [0..1]                               | ThrottleFactor        | `observingBrokerId`                           |
+| FallbackThrottleFactorApplied | Counter     | The number of times the plug-in has transitioned to using the fall back factor | ThrottleFactor        | `observingBrokerId`                           |
+| LimitViolated                 | Counter     | A count of the number `logDir`s which violate the configured limit             | ThrottleFactor        | `observingBrokerId`                           |
+| ActiveBrokers                 | Gauge       | The current number of brokers returned by the describeCluster rpc              | VolumeSource          | `observingBrokerId`                           |
+| ActiveLogDirs                 | Gauge       | The number of logDirs returned by the describeLogDirs RPC                      | VolumeSource          | `observingBrokerId`                           | 
+| AvailableBytes                | Gauge       | The number of available bytes returned by the describeLogDirs RPC              | VolumeSource          | `[observingBrokerId, remoteBrokerId, logDir]` |
+| ConsumedBytes                 | Gauge       | The number of consumed bytes returned by the describeLogDirs RPC               | VolumeSource          | `[observingBrokerId, remoteBrokerId, logDir]` |
```

Comment:
Now I see that tags like this it makes me wonder about the metric cardinality we have going on here. It's fine for smaller clusters, but it grows like $numBrokers^2$, so a 60 broker cluster would have 3,600 metrics for each of these.

Reply:
Hmm thats a valid point but probably not one to address within the scope of this PR.

Have raised it as https://github.com/strimzi/kafka-quotas-plugin/issues/43